variables.tf

```diff
@@ -30,6 +30,7 @@ variable "use_existing_resource_group" {
 variable "resource_group_name" {
   type        = string
   description = "The name of a new or an existing resource group where the resources are created."
+  default     = null
```

Comment:
why are you adding a default value? Isn't `resource_group_name` a required input?